ignite/services/chain/chain.go

```diff
@@ -444,14 +440,14 @@ func (c *Chain) Commands(ctx context.Context) (chaincmdrunner.Runner, error) {
 		return chaincmdrunner.Runner{}, err
 	}
 
-	config, err := c.Config()
+	cfg, err := c.Config()
 	if err != nil {
 		return chaincmdrunner.Runner{}, err
 	}
 
 	servers := chainconfigv1.DefaultServers()
-	if len(config.Validators) > 0 {
-		validator := config.Validators[0]
+	if len(cfg.Validators) > 0 {
+		validator, _ := chainconfig.FirstValidator(cfg)
```

Comment:
```suggestion
    validator, err := chainconfig.FirstValidator(cfg)
	if err == nil {		
```

Since the intent of the `FirstValidator()` function is to avoid checking the `Validators` slice, I would have written it like that.

Reply:
At some point I wrote it like that but I didn't commit it, arguably because of "semantics", checking length looks explicit, while `err == nil` might look that we are ignoring an error.

Reply:
OK for me!